lib/live_debugger_refactor/services/process_monitor/gen_servers/process_monitor.ex

```diff
@@ -0,0 +1,92 @@
+defmodule LiveDebuggerRefactor.Services.ProcessMonitor.GenServers.ProcessMonitor do
+  @moduledoc """
+  This module is monitoring the status of LiveView processes created by a debugged application.
+  """
+
+  use GenServer
+
+  alias LiveDebuggerRefactor.CommonTypes
+  alias LiveDebuggerRefactor.Bus
+  alias LiveDebuggerRefactor.Services.ProcessMonitor.Actions
+  alias LiveDebuggerRefactor.Services.CallbackTracer.Events.TraceCalled
+  alias LiveDebuggerRefactor.Services.CallbackTracer.Events.TraceReturned
```

Comment:
Could you make some order in these aliases - I'd like bus and events to be close together separated from other aliases
```suggestion
  alias LiveDebuggerRefactor.CommonTypes
  alias LiveDebuggerRefactor.Services.ProcessMonitor.Actions
  
  alias LiveDebuggerRefactor.Bus
  alias LiveDebuggerRefactor.Services.CallbackTracer.Events.TraceCalled
  alias LiveDebuggerRefactor.Services.CallbackTracer.Events.TraceReturned
```

Reply:
Also, I'd consider aliasing `ProcessMonitor.Actions` as `ProcessMonitorActions` like you did in tests